src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -173,8 +174,8 @@ public void updateFilteredPatientList(Predicate<Patient> predicate) {
     //=========== Room List ========================================================================================
 
     @Override
-    public int numOfOccupiedRooms() {
-        return roomList.numOfOccupiedRooms();
+    public int numOfExcessOccupiedRooms() {
```

Comment:
We may want to use verbs for the name of methods to make it clearer? `getNumOfOccupiedRooms()`?

May also want to make the Javadocs for this method clearer.